packages/widgets/src/lib/components/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -75,7 +81,21 @@
 		</div>
 	{/if}
 
-	{#if validExamples.length && applyWidgetExample}
-		<WidgetExamples {validExamples} {isLoading} {applyWidgetExample} {callApiOnMount} {exampleQueryParams} />
-	{/if}
+	<div class="flex gap-2 ml-auto">
+		{#if showReset}
```

Comment:
```suggestion
		{#if showReset && !isLoading && !isDisabled}
```

Reply:
Went with 
```svelte
{#if showReset && !isDisabled}
```

Because resetting input aborts the ongoing request and thus can work even if `isLoading`

wdyt?